### PR TITLE
fix(SL-1): Desktop nav styles broken

### PR DESCRIPTION
**Pull request by Alita-oss**

### CODE REVIEW

File: components/SHeader.vue [hunk 1]
```diff
@@ -128,11 +128,6 @@ header {
                     background-color: var(--color-green-600);
                 }
             }
-
-            &__link {
```
Comment (MarcusWiseman7) [inline components/SHeader.vue:132]: Why was this removed? Is it not needed anymore?

Comment (Alita-oss) [reply components/SHeader.vue:132]: It was a class that wasn´t used anywhere
